src/nvim/CMakeLists.txt

```diff
@@ -624,6 +625,7 @@ add_custom_command(
       ${LUA_FILETYPE_MODULE_SOURCE} "vim.filetype"
       ${LUA_FS_MODULE_SOURCE} "vim.fs"
       ${LUA_F_MODULE_SOURCE} "vim.F"
+      ${LUA_HELPTAGS_MODULE_SOURCE} "vim._helptags"
```

Comment:
not needed if it's moved to `_core` ... oh, i need to merge https://github.com/neovim/neovim/pull/35150 🙈